tests/jvm/src/test/resources/example/Traits_2.13.scala

```diff
@@ -6,7 +6,7 @@ trait T/*<=traits.T#*/ {
 
 sealed trait U/*<=traits.U#*/
 object U/*<=traits.U.*/ {
-  def u/*<=traits.U.u().*/: U/*=>traits.U#*/ = new /*<=local0*/U/*=>traits.U#*/ /*=>java.lang.Object#`<init>`().*/{}
+  def u/*<=traits.U.u().*/: U/*=>traits.U#*/ = new U/*=>traits.U#*/ /*=>java.lang.Object#`<init>`().*/{}
```

Comment:
here's your example with `new` not getting a local.

Reply:
Hmm... so this wasn't working for 2.13.x at all? Or is it only in this case with 2.13?

Reply:
It would probably be good to put that into synthetics :thinking: Could you put that into an issue, I can take a look later.